ios/UniversalAnalyticsPlugin.m

```diff
@@ -100,6 +100,23 @@ - (void) setAnonymizeIp: (CDVInvokedUrlCommand*)command
   [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void) setOptOut: (CDVInvokedUrlCommand*)command
+{
+  CDVPluginResult* pluginResult = nil;
+  NSString* optout = [command.arguments objectAtIndex:0];
+
+  if ( ! _trackerStarted) {
+    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Tracker not started"];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+    return;
+  }
+
+  [[GAI sharedInstance] setOptOut:[optout:boolValue]];
```

Comment:
I just tried it and it doesn't compile in IOS, but you can change two line for this:


[[GAI sharedInstance] setOptOut:optout];